<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarSocio</class>
 <widget class="QWidget" name="ModificarSocio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Socio</string>
  </property>
  <widget class="QLabel" name="labelNombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelApellidos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellidos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDNI">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDNI">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsuario">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTipo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
